src/spin_sdk/llm.py

```diff
@@ -12,14 +14,35 @@ class LLMInferencingParams:
     top_p: float = 0.9
     
 
-def generate_embeddings(model: str, text: Sequence[str]):
+def generate_embeddings(model: str, text: Sequence[str]) -> spin_llm.EmbeddingsResult:
+    """
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.llm(ErrorModelNotSupported)` will be raised if the component does not have access to the specified model.
```

Comment:
Sorry I didn't catch these earlier.  AFAICT, all of these error types (in this file and others) are malformed.  Instead of e.g. `spin_sdk.wit.types.Err(spin_sdk.wit.imports.llm(ErrorModelNotSupported)` it should be `spin_sdk.wit.types.Err(spin_sdk.wit.imports.llm.ErrorModelNotSupported)`